build.gradle

```diff
@@ -261,6 +277,10 @@ dependencies {
   // XML
   compile 'javax.xml.bind:jaxb-api:2.3.0'
 
+  // database
+  compile 'org.apache.openjpa:openjpa:3.1.0'
```

Comment:
Couldn't all these dependency version be in `versions.gradle`?

Reply:
Done.
Didn't exist, but it does now